src/Simplify_Call.cpp

```diff
@@ -362,6 +362,18 @@ Expr Simplify::visit(const Call *op, ExprInfo *bounds) {
             return mutate(unbroadcast, bounds);
         }
 
+        if (bounds) {
+            bounds->min = 0;
+            bounds->min_defined = true;
+            if (a_bounds.min_defined && a_bounds.max_defined) {
+                if (a_bounds.min != std::numeric_limits<int64_t>::min() &&
+                    a_bounds.max != std::numeric_limits<int64_t>::min()) {
```

Comment:
When would a.bounds.max ever equal numeric_limits::min (vs max)?

Reply:
A constant equal to `t = std::numeric_limits<int64_t>::min()` would have `min = max = t`

Reply:
ahh, gotcha, didn't think about the constant case